docs/source/cause_model_impact.rst

```diff
@@ -0,0 +1,53 @@
+===============================
+Vivarium Research Documentation
+===============================
+
+Mode impact 
+-------------
+
+1. **What is model impact**
+ Mode impact is about how causes affect mortality/disability
+2. **Mortality**
+
+ image:: hazard_ratio.png
+  
+
+ - Survival analysis: what is the probability that the time of death 
+   is between now and the end of this time step, given that time of 
+   death later than now. 
+
+   morality rate :math:`\lambda=P(t < T_{death} <= t + \triangle_{t} | T_{death} > t) = 1 - e^{-\lambda\triangle t}`
+   
+
+ - In simulation, we take a uniform random number evenly distributed 
+   between zero and one. If that number less than the probability, it 
+   returns death. 
+   
+   answer: yes if :math:`u<= 1 - e^{-\lambda\triangle t}` else no
+          
+   where :math:`u \rightarrow  U[0,1]`
+   
+ - Use GBD data to estimate mortality rate (three cases)
+
+   case 1: no cause models
+    :math:`\lambda=ACMR`
+   case 2: one cause model
+    :math:`\lambda = ACMR - CSMR + EMR(disease\_status)`
+   case 3: multiple causes model 
+    :math:`\lambda_{i} = ACMR - \sum_{diseases}CSMR_{c} + \sum_{diseases}EMR_{c}(disease\_status_{c})`
+   what did the person die of
+    :math:`P_{c} = \frac{EMR_{c}}{CDMR + \sum_{diseases}EMR_{c}(disease\_status_{c})}`
```

Comment:
I think we should to define the acronyms ACMR, CSMR, and EMR

Reply:
Is it okay that I add following annotation afterwards:
All cause mortality rate (ACMR)
Cause specific mortality rate (CSMR)
Excess mortality rate (EMR)

Reply:
Yeah, I think that's perfect. Thanks :)